jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/task/SynchronizedTaskService.java

```diff
@@ -543,6 +543,13 @@ public void suspend(long taskId, String userId) {
         }
     }
```

Comment:
probably the method with 2 params is never called now

Reply:
need to keep this for backward compatibility